packages/playwright-test/src/reporters/raw.ts

```diff
@@ -317,4 +317,12 @@ function dedupeSteps(steps: JsonTestStep[]): JsonTestStep[] {
   return result;
 }
 
+function filterOutPrivateFields(object: any): any {
+  if (!object || typeof object !== 'object')
+    return object;
+  if (Array.isArray(object))
+    return object.map(filterOutPrivateFields);
+  return Object.fromEntries(Object.entries(object).filter(entry => !entry[0].startsWith('_')).map(entry => [entry[0], filterOutPrivateFields(entry[1])]));
+}
```

Comment:
Let's make sure this change lands regardless of the fate of plugins.

Reply:
I think we'll just _plugins